src/bridge/MatrixHandler.ts

```diff
@@ -1097,7 +1097,7 @@ export class MatrixHandler {
 
     private async textForReplyEvent(event: MatrixMessageEvent, replyEventId: string, ircRoom: IrcRoom):
     Promise<{formatted: string; reply: string}|null> {
-        const REPLY_REGEX = /> <(@.*:.*)>(.*)\n\n(.*)/;
+        const REPLY_REGEX = /> <(.*)>(.*)\n\n(.*)/;
```

Comment:
Might want to change this to:

```js
const REPLY_REGEX = /> <(.*?)>(.*?)\n\n(.*)/;
```

These [non-greedy matches](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Quantifiers) make sure that in e.g.

```
<SomeName> if x > 2

Some reply that could, technically

Contain >2 newlines

Or even more
```

the first group will match `SomeName` instead of `SomeName> if x`, or, in this case,

```
SomeName> if x > 2

Some reply that could, technically,

Contain 
```

And the second group `if x > 2` instead of `  2` or, in this case, `2 newlines`,
and the third group 

```
Some reply that could, technically

Contain >2 newlines

Or even more
```

instead of 

```
Contain >2 newlines

Or even more
```

or, in this case,

```
Or even more
```

Reply:
I just realized that this *would* break on display names with `>` in them. To rule this out, I guess you'd need to parse the HTML.